locales/en.yml

```diff
@@ -1,5 +1,21 @@
 ---
 en:
+  about:
```

Comment:
It might be worth keeping your eye on the size of this data file. It might get unwieldy as you add more to it. Or maybe that's the way one does it in Middleman?

Reply:
You suggesting breaking it into multiple files? I believe I can do that in Middleman

Reply:
I'd say `keep your eye on it`.

When you start building entire pages, like a portfolio entry, you might find this file getting more difficult to reason about and update.

Reply:
Cool - yeah this site will get text heavy - I'll utilize the data files feature